Content/ansible/07-templates.md

```
@@ -0,0 +1,355 @@
+## What are Templates?
+
+**In short**: A template is a dynamic file - a file that contains variables which are replaced with actual values at runtime.
+
+Instead of writing:
+```plaintext
+server_name example.com;
+port 80;
+```
+
+You write a template:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)